autogpt_platform/backend/backend/blocks/wavespeed/ai_media_generator.py

```
@@ -0,0 +1,202 @@
+        submit_body: dict[str, Any] = {
+            **input_data.extra_inputs,
+            "prompt": input_data.prompt,
+        }
+
+        try:
+            submit_response = await Requests().post(
+                submit_url, headers=headers, json=submit_body
+            )
+            submit_data = submit_response.json()
+        except ClientResponseError as e:
+            logger.error(f"WaveSpeed API request failed: {str(e)}")
```

Ruff (0.16.1)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)